docs/en/04_Changelogs/4.11.0.md

```diff
@@ -21,6 +23,21 @@ In accordance with our [PHP support policy](/Getting_Started/Server_Requirements
 
 ## Features and enhancements {#features-and-enhancements}
 
+### Preview any DataObject in any admin section {#cms-preview}
+
+The CMS preview panel has historically on been available on `Versioned` `DataObjects` in the Pages admin section. This has now been expanded to allow any `DataObject`(regardless of whethwr they jse the`Versioned` extension) to be previewed in any section of the CMS.
```

Comment:
```suggestion
The CMS preview panel has historically on been available for `Versioned` `DataObjects` in the Pages admin section. This has now been expanded to allow any `DataObject`(regardless of whether they use the`Versioned` extension) to be previewed in any section of the CMS.
```

Reply:
You know it's your PR so you can update these straight away, right? :-D

Reply:
I made this PR and found those on my tablet. I don't wanna make a bunch of little "fixed typo" commits and I don't have a way of doing a `git commit -amend` and `git force --force` without turning on my computer but my couch is so comfy :p